src/vm/eetwain.cpp

```diff
@@ -3805,6 +3806,28 @@ bool UnwindEbpDoubleAlignFrame(
     {
         TADDR baseSP;
 
+#ifdef WIN64EXCEPTIONS
+        // Funclets' frame pointers(EBP) are always restored so they can access to main function's local variables.
+        // Therefore the value of EBP is invalid for unwinder so we should use ESP instead.
+        // If funclet frame layout is changed from CodeGen::genFuncletProlog() and genFuncletEpilog(),
+        // we need to change here accordingly.
+        if (pCodeInfo->IsFunclet())
+        {
+            baseSP = curESP + 8; // padding for 16byte stack alignment allocated in genFuncletProlog()
```

Comment:
@janvorli Thank you for explanation. Is funclet body (especially function call sequnece in it) also non-interruptible? If not, it seems that GC would get stuck for the following case (with the above unwind routine):
```
push STK_ARG_2
<< GC >>
push STK_ARG _1
call FUNC
```
As I understand, x86/Windows also seems to get stuck for the above case as unwinder couldn't figure out where saved ``ebp`` is, but it seems to work. That is an interesting point for me.